modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -791,85 +789,85 @@ def obj.class
           end
         end
       end
+    end
 
-      describe "'disabilities.ratedDisabilityId' validations" do
```

Comment:
I noticed that these `disabilities.ratedDisabilityId` specs I recently added accidentally got enclosed in another context/scope.  So I moved them out to the appropriate level.